### PR TITLE
Apple Clone

### PR DESCRIPTION
**Pull request by haraphat01**
This is the clone of Apple's Website Page. This page is built with just Html and CSS. Some amazing it has is:

- Usage of Grid and Flex

- Usage of Gradient Background and also,

- Usage of the background picture.

### CODE REVIEW

**Comment by uryelah** [comment]
Hello **Arafat Akata**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**Good job working with background images! 🌟**

There are still some required checks you need to fulfill in order to continue to the next project, so hang in there! 💪

---------     

## Required Changes 🏗 👷‍♀ 

- [ ] Make sure there are no warning in yout HTML validation at https://validator.w3.org/nu.

- [ ] All but the last navbar element should be links. Make sure the apple logo is also a link.

- [ ] Make sure there are no unnecessary borders around your page elements.
- [ ] The main image should take 100% of the screen width regardless of its size.

Compare your clone and the original site in the screenshots below:
**Clone:**
Screenshot_2020-04-13 Apple Clone

**Original site:**
Screenshot_2020-04-13 Apple(1)

-----

- [ ] The elements at the bottom of the main image should be links, notice how the link you to other pages in the[ original website](https://web.archive.org/web/20140301004610/http://www.apple.com/) Check the requirement below and make sure to fulfill it. 
Screenshot_2020-04-13 dist html_css building_with_backgrounds_and_gradients md · master · Microverse guides tse code_review

----

- [ ] [OPTIONAL] Be more semantic! Use the `main` tag for the first section and `aside` for the second one with the bottom images.

If the page looks correct on your monitor you can use your [browser responsive tools](https://developer.mozilla.org/en-US/docs/Tools/Responsive_Design_Mode) to make sure the page still looks correct on slightly wider monitors.

------

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by haraphat01** [comment]
The changes requested  has been effected in the following areas;

- No more warning sign from Html Validator

- Apple logo is now a link

- All unnecessary borders removed

- The images now take 100% of its width container

- The main page elements are now links

**Comment by sumancrest0001** [comment]
**STATUS: CHANGES REQUIRED
Hello @haraphat01** 
Nice work :clap: :clap: 
Your page layout is good but we have to make sure that it looks like the original one. We have to fix a couple of issues.
[Required] Please add the stickler file. It seems you forgot to add the .stickler.yml file.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+          </div>
+        </section>
+
+        <aside>
+          <div class="promo-section">
+            <div class="iphone-5s"></div>
+            <div class="iphone-5c"></div>
+            <div class="birthday"></div>
+            <div class="supplier"></div>
+          </div>
```
Comment (sumancrest0001) [inline index.html]: [Require] Add links to these articles.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+  height: 100%;
+  width: 100%;
+  background-color: white;
+  font-family: "Raleway", sans-serif;
+}
+
+header {
+  width: 70%;
+  background: url("./images/background.jpg") no-repeat;
+
```
Comment (sumancrest0001) [inline main.css]: [Required] width of 70% did not allow you page to cover the whole screen. Please change it.

**Comment by sumancrest0001** [comment]
[Required] Please reduce height of the vertical lines at the footer section.

**Comment by sumancrest0001** [comment]
**Kindly make the above-requested changes and submit the project for another review.
Happy coding  :slightly_smiling_face:** 
Need help feel? message me on slack **Suman Shrestha**

**Comment by haraphat01** [comment]
Thank you, TSE for the amazing job. As the necessary modifications have been applied to this.

- The width is now 100%.

- CSS linter errors have been corrected

- The vertical line section has been changed too

- Links added to the article section.

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi @haraphat01 👋

You done quite well in this project.
Your navbar background color is really impressive👍👏

However, kindly make the below changes to make your project look even more awesome:

- [ ] Required - Kindly take a look at both screenshots below, observe the differences and make the changes that follow.
**THIS IS YOURS👇**
ice_screenshot_20200414-132232

**THIS IS THE WAY IT SHOULD BE👇**
ice_screenshot_20200414-133734

Requested Changes:
* KIndly change the background-position of the main section image, as can be observed, yours is quite at the lower portion of the image, opposed to the original webpage where it is centered.

* Also increase the text sizes of the three centered text and consider placing them a bit lower on the page.

- [ ] Required - Your navbar looks quite different from the original web page. Please adjust the position of the search icon and kindly remove the white border around the search input field.
**THIS IS YOURS👇**
ice_screenshot_20200414-132311

**THIS IS THE WAY IT SHOULD BE👇**
ice_screenshot_20200414-135215


- [ ] Required - In addition kindly increase the text size of the footer text elements, as they look quite smaller than the original webpage footer texts.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**Comment by haraphat01** [comment]
Thank you for the review. The following adjustment has been made;

- The position of the image has been changed.

- The font size of the paragraph  increased

- Changed on the search bar too 

- The footer has been reworked on too.

**Comment by christianotieno** [comment]
## Pull Request under review by Christian Otieno (TSE)

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project @haraphat01. :fireworks: 
##
***All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.***
- Congratulations! :+1:
- You can now merge to master and I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**